nodes/core/core/20-inject.html

```diff
@@ -165,7 +166,7 @@ <h3>Details</h3>
 <p>The node also supports injecting strings, numbers, booleans, JavaScript objects, or flow/global context values.</p>
 <p>By default, the node is triggered manually by clicking on its button within the editor. It can also be set to
 inject at regular intervals or according to a schedule.</p>
-<p>It can also be configured to inject once each time the flows are started.</p>
+<p>It can also be configured to inject once with a delay each time if the start of flows occurred on deploy.</p>
```

Comment:
not sure why you are re-wording this ? What is the clarification you are trying to make ? (obviously I'm not getting it any clearer :-)

Reply:
set back to original text